docs/user-guide/rules/html-checker.md

```diff
@@ -0,0 +1,68 @@
+# The Nu HTML Test (`html-checker`)
+
+`html-checker` validates the markup of a website against the [Nu HTML checker](https://validator.github.io/validator/).
+
+## Why is this important?
+
+> Serving valid HTML nowadays have been commonly overlooked these days.
+By running the HTML documents through a checker, it's easier to catch
+unintended mistakes which might have otherwise been missed.
+Adhering to the W3C' standards has a lot to offer to both the
+developers and the web users: It provides better browser compatibility,
+helps to avoid potential problems with accessibility/usability, and makes it easier for future maintainance.
```

Comment:
Were possible, try to keep the text under 80 characters / line. The same goes for the lines below.